lib/validator.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class Validator
+  VALID_YES_NO_REPLY = /^[yn]$/i.freeze
+
+  def valid_choice?(option)
+    option.match?(/^[1-2]$/)
```

Comment:
I would leave this knowledge with the UI as you had it before.
It's not the responsibility of the validator to know how many choices the menu has.